examples/seq2seq/finetune.sh

```diff
@@ -5,10 +5,9 @@ export PYTHONPATH="../":"${PYTHONPATH}"
 python finetune.py \
     --learning_rate=3e-5 \
     --fp16 \
-    --gpus 1 \
+    --gpus 2 \
```

Comment:
this shouldn't go in finetune.sh. You can make a new finetune_multigpu.sh if you'd like